src/service/resources-impl.js

```diff
@@ -771,10 +777,16 @@ export class Resources {
           resize = true;
         } else if (diff < 0) {
           // 6. The new height is smaller than the current one.
+          if (request.callback) {
+            request.callback(false);
+          }
           resize = false;
         } else {
           // 7. Element is in viewport don't resize and try overflow callback
           // instead.
+          if (request.callback) {
+            request.callback(false);
+          }
           request.resource.overflowCallback(/* overflown */ true,
```

Comment:
No it should not be. because - ads can still have an overflow element and this still needs to be called. 
`overflowCallback` has only been removed from the `BaseElement` and is still there in the `CustomElement`